index.html

```diff
@@ -47,13 +47,15 @@ <h3>Generic Problem Description</h3>
       <section id="use-case-1">
         <h3>Use-case #1: Cross-App Communications</h3>
```

Comment:
This line should also be changed, if use-case 1 were to dive deeply into the case of VC/Slides, rather than use it as an example (which was the intention with the original text).

Reply:
That's https://github.com/WICG/capture-handle/pull/7.